content/sensu-go/5.19/sensuctl/reference.md

```diff
@@ -21,7 +21,8 @@ menu:
 - [Export resources](#export-resources)
 - [Manage resources](#manage-resources)
   - [Subcommands](#subcommands)
-- [Response filtering](#response-filtering) (commercial feature)
+- [Prune resources](#prune-resources) (experimental feature)
+- [Response filtering](#response-filtering)
```

Comment:
I'm not sure if you meant to remove `(commercial feature)` from response filtering here?

Reply:
Yeah, I should have removed it sooner. We have the commercial features note in the response filtering text. Thank you for checking.